docs/configure-ems.md

```diff
@@ -111,7 +111,8 @@ The EMS event template parameters are explained below along with an example for
       named `LUN.online` is received.
     - `resolve_after` (optional, Go duration, default = 28 days) resolve the issuing EMS after the specified duration
       has elapsed (`672h` = `28d`).
-      If the bookend pair is not received within the `resolve_after` duration, the issuing EMS event expires.
+      If the bookend pair is not received within the `resolve_after` duration, the Issuing EMS event expires. Harvest
```

Comment:
Maybe this

If the bookend pair is not received within the `resolve_after` duration, the issuing EMS event expires. When that happens, Harvest will mark the event as auto resolved by adding the `autoresolve=true` label to the issuing EMS event.